pupil_src/shared_modules/video_capture/file_backend.py

```diff
@@ -82,16 +80,16 @@ class FakeFrame:
     gap between timestamp.
     """
 
-    static_img = np.ones((720, 1280, 3), dtype=np.uint8) * 128
-
-    def __init__(self, timestamp, index):
+    def __init__(self, shape, timestamp, index):
+        self.shape = shape
         self.yuv_buffer = None
-        self.img = self.bgr = self.gray = self.static_img.copy()
+        static_img = np.ones(self.shape, dtype=np.uint8) * 128
+        self.img = self.bgr = self.gray = static_img.copy()
```

Comment:
There is no need to create a copy here since `static_img` is newly instantiated everytime.